src/main/java/org/humancellatlas/ingest/process/ProcessService.java

```diff
@@ -99,32 +112,16 @@ private File determineTargetFile(SubmissionEnvelope submissionEnvelope, File fil
         return targetFile;
     }
 
-    public Process resolveBundleReferencesForProcess(final Process analysis, BundleReference bundleReference) {
+    public Process addInputBundleManifest(final Process analysisProcess, BundleReference bundleReference) {
         for (String bundleUuid : bundleReference.getBundleUuids()) {
             Optional<BundleManifest> maybeBundleManifest = getBundleManifestRepository().findTopByBundleUuidOrderByBundleVersionDesc(bundleUuid);
```

Comment:
we might want the greenbox to just give us the specific bundle version.

Reply:
Yeah with this endpoint and the other we should look for a version as UUID is now multi version